index.bs

```diff
@@ -379,6 +379,34 @@ and on when to use promises and when not to use promises,
 see <strong><a href="https://www.w3.org/2001/tag/doc/promises-guide">Writing
 Promise-Using Specifications</a></strong>.
 
+<h3 id="aborting">Cancel asynchronous APIs/operations using AbortSignal</h3>
+
+Async functions that need cancellation should take an `AbortSignal` as part
+of an options dictionary.
+
+Example:
+
+```js
+const controller = new AbortController();
+const signal = controller.signal;
+geolocation.read({ signal });
+```
+
+Reusing the same primitive everywhere has multiplicative effects throughout
+the platform. In particular, there's a common pattern of using a single
+`AbortSignal` for a bunch of ongoing operations, and then aborting them
+(with the corresponding `AbortController`) when e.g. the user presses cancel,
+or a single-page-app navigation occurs, or similar. So the minor extra
+complexity for an individual API leads to a large reduction in complexity
+when used with multiple APIs together.
+
+There might be cases where cancellation cannot be guaranteed. That is not a
+problem because a call to `abort()` on `AbortController` is a request to abort.
+How you react to it depends on your spec.
+
+<p class="note">Note that we could have given it a verbose name like
```

Comment:
Maybe instead: "Note, `requestAbort()` was considered in the AbortController design instead of `abort()`, but the latter was chosen for brevity."

I don't think this needs to be in a new paragraph.. just tack it on to the end of the last paragraph without the class="note" styling ;)